src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-inline: 3px;
+    display: block;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the image should be centered horizontally. Since you're using `margin-inline: 3px;`, it may not center the image properly if the `.card` width changes. Consider using `margin: 0 auto;` to ensure the image remains centered.